src/Lib/globals.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (C) 2024 NetPanzer (https://github.com/netpanzer/), Andy Alt, Et al.
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+*/
+
+#include "globals.hpp"
+
+#ifndef PACKAGE_VERSION
+#include "config.h"
+#endif
+
+const std::string Package::GetVersion(void)
```

Comment:
sorry - can we rename this to package.cpp/hpp now? worried people will try to treat globals like "utils" :P

Reply:
> sorry - can we rename this to package.cpp/hpp now? worried people will try to treat globals like "utils" :P

No apology necessary. Done.

Let's remember to "squash and merge" this PR when merging.